src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelGetSemanticInfoTests.cs

```diff
@@ -15213,5 +15213,24 @@ class K
 
             Assert.False(semanticInfo.IsCompileTimeConstant);
         }
+
+        [Fact]
```

Comment:
Please add `[WorkItem(18763, "https://github.com/dotnet/roslyn/issues/18763")]`. (I should have included that originally.)